hasher-matcher-actioner/hmalib/lambdas/api/api_root.py

```diff
@@ -114,46 +93,6 @@ class SignalSourceSummary(t.TypedDict):
     updated_at: str
 
 
-def get_signals() -> t.List[SignalSourceSummary]:
```

Comment:
This breaks the signal page, if you want to go through with this change and not update `"/signals"` please also delete the endpoint, `webapp/src/pages/Signals.jsx`, and its associated sidebar item.

Reply:
Thanks for the flag. Already prepping a PR for removing the signals sidebar.

I'll drop the new one on your plate so we can merge both in together and not break master.

Reply:
stamped then.